src/content/protocols/compound-v3.md

```diff
@@ -0,0 +1,154 @@
+| Proxy Admin                 | renounceOwnership                     | Abandons ownership of the contract. The DAO would renounce the access to the administrative functions of the contracts, which includes upgrading the `Comet Implementation` and `Configurator Implementation` contracts.        | Timelock (DAO)                 |
+| Proxy Admin                 | transferOwnership                     | Updates the owner of the `ProxyAdmin` contract: the account with the rights to change the admin of the proxy and upgrade the implementation contracts.                                                                          | Timelock (DAO)                 |
+| Proxy Admin                 | deployAndUpgradeTo                    | Deploy a new Comet and upgrade the implementation of the Comet proxy                                                                                                                                                            | Timelock (DAO)                 |
+| Proxy Admin                 | deployUpgradeToAndCall                | Deploy a new Comet and upgrade the implementation of the Comet proxy, then call a function in the new contract.                                                                                                                 | Timelock (DAO)                 |
+| Rewards                     | setRewardConfig                       | Set the reward token for a Comet instance.                                                                                                                                                                                      | Timelock (DAO)                 |
+| Rewards                     | withdrawToken                         | Withdraw reward tokens from the contract.                                                                                                                                                                                       | Timelock (DAO)                 |
+| Rewards                     | transferGovernor                      | Transfers the governor rights to a new address.                                                                                                                                                                                 | Timelock (DAO)                 |
+| Rewards                     | claimTo                               | Claim rewards from a comet instance to a target address .                                                                                                                                                                       | Timelock (DAO)                 |
+| Governor                    | \_setImplementation                   | Updates the implementation of the `Governor` contract. Can only be triggered by the DAO itself. The new contract would inherit all the DAO permissions mentioned above.                                                         | Timelock (DAO)                 |
```

Comment:
replace Governor with `GovernorBravoDelegator` and add subsequent rows for permissions owned by the DAO specified in the Implementation contract (GovernorBravoDelegator is the proxy)

see here: https://www.defiscan.info/protocols/uniswap-v2-ethereum